### PR TITLE
GEODE-7783: Introduce dispersion in connection expiration

### PR DESCRIPTION
**Pull request by alb3rtobr**

### CODE REVIEW

File: cppcache/src/TcrConnection.hpp [hunk 1]
```diff
@@ -142,7 +142,23 @@ class APACHE_GEODE_EXPORT TcrConnection {
...
         m_isUsed(0),
-        m_poolDM(nullptr) {}
+        m_poolDM(nullptr) {
+    auto nowTimePoint = clock::now().time_since_epoch();
+    auto now_ms =
+        std::chrono::duration_cast<std::chrono::milliseconds>(nowTimePoint)
+            .count();
+    auto now_s =
+        std::chrono::duration_cast<std::chrono::seconds>(nowTimePoint).count();
+    srand((now_s * 1000) + (now_ms / 1000));
```
Comment (pdxcodemonkey) [inline cppcache/src/TcrConnection.hpp]: This introduces a build break on Windows.  Will build it locally today and post a link to a fix.

Comment (pdxcodemonkey) [reply cppcache/src/TcrConnection.hpp]: This commit fixes the build issue:  https://github.com/pdxcodemonkey/geode-native/commit/ebef1e64f004453f69aa2e743ad98d26f583de8d.  Please incorporate it or something similar, then I believe you're good to go.

Comment (alb3rtobr) [reply cppcache/src/TcrConnection.hpp]: thanks for the patch!

Comment (alb3rtobr) [reply cppcache/src/TcrConnection.hpp]: done, I had to change it a bit, but it seems its fine

**Pull request merged**